docs/release-notes.md

```diff
@@ -1,9 +1,50 @@
 # Release notes
 
-## Trino Gateway 10 (24 July 2024)
+## Trino Gateway 11 (5 Sep 2024)
+
+[JAR file gateway-ha-11-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/11/gateway-ha-11-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:11`
+
+* [:warning: Breaking change:](#breaking) Require Java 22 for build and runtime.
+  ([#441](https://github.com/trinodb/trino-gateway/pull/441))
+* Add support for using an external service with a REST API to process requests
+  and implement any custom routing rules.
+  ([#423](https://github.com/trinodb/trino-gateway/pull/423))
+* Add support to optionally deactivate adding `X-Forwarded-*` HTTP headers to
```

Comment:
Can we make this a bit more user facing somehow @willmostly

Reply:
how about

Add an option to forward requests without adding `X-Forwarded-*` HTTP headers, which will cause the client to connect directly to the coordinator running the query. 
([#417](https://github.com/trinodb/trino-gateway/pull/417))

Reply:
slightly adapted